.nyc_output/427d34dc-25e9-45b0-ab1a-032c7a7d7f38.json

```diff
@@ -0,0 +1 @@
11}},"line":221},"31":{"name":"(anonymous_31)","decl":{"start":{"line":229,"column":25},"end":{"line":229,"column":26}},"loc":{"start":{"line":229,"column":41},"end":{"line":231,"column":9}},"line":229},"32":{"name":"(anonymous_32)","decl":{"start":{"line":231,"column":20},"end":{"line":231,"column":21}},"loc":{"start":{"line":231,"column":35},"end":{"line":233,"column":9}},"line":231},"33":{"name":"(anonymous_33)","decl":{"start":{"line":241,"column":31},"end":{"line":241,"column":32}},"loc":{"start":{"line":241,"column":45},"end":{"line":254,"column":1}},"line":241},"34":{"name":"(anonymous_34)","decl":{"start":{"line":245,"column":21},"end":{"line":245,"column":22}},"loc":{"start":{"line":245,"column":48},"end":{"line":253,"column":3}},"line":245},"35":{"name":"(anonymous_35)","decl":{"start":{"line":248,"column":12},"end":{"line":248,"column":13}},"loc":{"start":{"line":248,"column":24},"end":{"line":250,"column":5}},"line":248},"36":{"name":"(anonymous_36)","decl":{"start":{"line":250,"column":16},"end":{"line":250,"column":17}},"loc":{"start":{"line":250,"column":31},"end":{"line":252,"column":5}},"line":250},"37":{"name":"(anonymous_37)","decl":{"start":{"line":256,"column":32},"end":{"line":256,"column":33}},"loc":{"start":{"line":256,"column":54},"end":{"line":282,"column":1}},"line":256},"38":{"name":"(anonymous_38)","decl":{"start":{"line":260,"column":21},"end":{"line":260,"column":22}},"loc":{"start":{"line":260,"column":48},"end":{"line":281,"column":3}},"line":260},"39":{"name":"(anonymous_39)","decl":{"start":{"line":276,"column":12},"end":{"line":276,"column":13}},"loc":{"start":{"line":276,"column":29},"end":{"line":278,"column":5}},"line":276},"40":{"name":"(anonymous_40)","decl":{"start":{"line":278,"column":16},"end":{"line":278,"column":17}},"loc":{"start":{"line":278,"column":31},"end":{"line":280,"column":5}},"line":278},"41":{"name":"(anonymous_41)","decl":{"start":{"line":284,"column":33},"end":{"line":284,"column":34}},"loc":{"start":{"line":284,"column":47},"end":{"line":295,"column":1}},"line":284},"42":{"name":"(anonymous_42)","decl":{"start":{"line":288,"column":21},"end":{"line":288,"column":22}},"loc":{"start":{"line":288,"column":48},"end":{"line":294,"column":3}},"line":288},"43":{"name":"(anonymous_43)","decl":{"start":{"line":289,"column":24},"end":{"line":289,"column":25}},"loc":{"start":{"line":289,"column":40},"end":{"line":291,"column":5}},"line":289},"44":{"name":"(anonymous_44)","decl":{"start":{"line":291,"column":16},"end":{"line":291,"column":17}},"loc":{"start":{"line":291,"column":31},"end":{"line":293,"column":5}},"line":291},"45":{"name":"(anonymous_45)","decl":{"start":{"line":297,"column":32},"end":{"line":297,"column":33}},"loc":{"start":{"line":297,"column":67},"end":{"line":357,"column":1}},"line":297},"46":{"name":"(anonymous_46)","decl":{"start":{"line":304,"column":21},"end":{"line":304,"column":22}},"loc":{"start":{"line":304,"column":48},"end":{"line":356,"column":3}},"line":304},"47":{"
```

Comment:
Pretty sure this directory should be added to `.gitignore`. Those `.eslintrc.json.swp` and `.swo` above might also be for ignoring? Not really sure what they do.

Reply:
Yup good catch